avae/napari.py

```diff
@@ -159,10 +159,7 @@ def _load_data(self):
     def add_pose_widget(self) -> None:
         """Add widgets to manipulate the model pose space."""
         pose_axes = QtWidgets.QComboBox()
-        axis = ["X", "Y", "Z"]
-
-        for dim in range(self.pose_dims):
-            pose_axes.addItems([axis[dim]])
+        pose_axes.addItems(["X", "Y", "Z"])
```

Comment:
If we are not using specifically the differentiable model this will break the napari plugin. The previous implementation should work for both.

Reply:
resolved in https://github.com/alan-turing-institute/affinity-vae/pull/280/commits/aa4fa25beca0be0e4f592cb19ab92908eb0898d2